.travis.yml

```diff
@@ -16,6 +16,7 @@ jobs:
 
       before_install:
         - composer validate
+        - composer normalize --dry-run
```

Comment:
Probably doesn't make sense to add this to the `stan` stage, but I don't think that running

```
$ composer normalize --dry-run
```

is worth a stage of its own.

Perhaps we add `friendsofphp/php-cs-fixer` as a development dependency in a separate PR, and then run both in a `style` stage?